day2OfJS/day2.js

```diff
@@ -0,0 +1,234 @@
+//15. Use indexOf to determine the position of the first occurrence of a in 30 Days Of JavaScript
+const occurrenc_e = '30 Days Of JavaScript';
+console.log(occurrenc_e.indexOf('a')); //4
+
+
+//16. Use lastIndexOf to determine the position of the last occurrence of a in 30 Days Of JavaScript.
+const occurrenc_e2 = '30 Days Of JavaScript';
+console.log(occurrenc_e2.lastIndexOf('a')); //14
+
+
+//17. Use indexOf to find the position of the first occurrence of the word because in the following sentence:'You cannot end a sentence with because because because is a conjunction'
+const occurrenc_e3 = 'You cannot end a sentence with because because because is a conjunction';
+console.log(occurrenc_e3.indexOf('because')); //31
+
+
+//18. Use lastIndexOf to find the position of the last occurrence of the word because in the following sentence:'You cannot end a sentence with because because because is a conjunction'
+const occurrenc_e4 = 'You cannot end a sentence with because because because is a conjunction';
+console.log(occurrenc_e4.lastIndexOf('because')); //47
+
+
+//19. Use search to find the position of the first occurrence of the word because in the following sentence:'You cannot end a sentence with because because because is a conjunction'
+const occurrenc_e5 = 'You cannot end a sentence with because because because is a conjunction';
+console.log(occurrenc_e5.search('because')); //31
+
+
+//20. Use trim() to remove any trailing whitespace at the beginning and the end of a string.E.g ' 30 Days Of JavaScript '.
+const remov_e = ' 30 Days Of JavaScript ';
+console.log(remov_e.trim(' 30 Days Of JavaScript ')); //30 Day Of JavaScript
+
+
+//21. Use startsWith() method with the string 30 Days Of JavaScript and make the result true
+const strin_g = '30 Days Of JavaScript';
+console.log(strin_g.startsWith('30')); //true
+
+
+//22. Use endsWith() method with the string 30 Days Of JavaScript and make the result true
+const strin_g2 = '30 Days Of JavaScript';
+console.log(strin_g2.endsWith('Script')); //true
+
+
+//23. Use match() method to find all the a’s in 30 Days Of JavaScript
+const strin_g3 = '30 Days Of JavaScript';
+console.log(strin_g3.match(/a/gi)); //[ 'a', 'a', 'a' ]
+
+
+//24. Use concat() and merge '30 Days of' and 'JavaScript' to a single string, '30 Days Of JavaScript'
+const join = '30 Days Of';
+console.log(join.concat(' JavaScript')); //30 Days Of JavaScript
+
+
+//25. Use repeat() method to print 30 Days Of JavaScript 2 times
+const phrase2 = '30 Days Of JavaScript';
+console.log(phrase2.repeat(2)); //30 Days Of JavaScript30 Days Of JavaScript
+
+
+
+
+/*
+Exercise: Level 2
+1. Using console.log() print out the following statement:
+The quote 'There is no exercise better for the heart than reaching down and lifting people up.' by John Holmes teaches us to help one another.
+*/
+console.log("The quote 'There is no exercise better for the heart than reaching down and lifting people up.' by John Holmes teaches us to help one another.");
```

Comment:
Please, avoid doing all the logic inside the console.log statement, instead assign the result of the logic to a variable and console.log the variable.